package.json

```diff
@@ -12,8 +12,8 @@
     "grunt-eslint": "^13.0.0",
     "grunt-requirejs": "^0.4.2",
     "mobify-code-style": "^2.2.1",
-    "nightwatch": "0.7.9",
-    "nightwatch-commands": "1.1.1"
+    "nightwatch": "0.8.16",
```

Comment:
Nope, we thought that 0.8.16 might have a bug but I couldn't reproduce it in other projects.